Frameworks/CoreGraphics/CGBitmapImage.mm

```diff
@@ -452,7 +454,7 @@
         BYTE* imageData = static_cast<BYTE*>(LockImageData());
         ComPtr<IWICBitmap> wicBitmap = Make<CGIWICBitmap>(this, SurfaceFormat());
         ComPtr<ID2D1Factory> d2dFactory;
-        THROW_IF_FAILED(D2D1CreateFactory(D2D1_FACTORY_TYPE_SINGLE_THREADED, __uuidof(ID2D1Factory), &d2dFactory));
+        FAIL_FAST_IF_FAILED(_CGGetD2DFactory(&d2dFactory));
         ComPtr<ID2D1RenderTarget> renderTarget;
         THROW_IF_FAILED(d2dFactory->CreateWicBitmapRenderTarget(wicBitmap.Get(), D2D1::RenderTargetProperties(), &renderTarget));
```

Comment:
We should sync up.

---
In reply to: [86614591](https://github.com/Microsoft/WinObjC/pull/1307#discussion_r86614591) [](ancestors = 86614591,86612430)